src/os/osEPiWriteIo.c

```diff
@@ -0,0 +1,15 @@
+#include "common.h"
+
+#ifdef MOVE_ISSUE
```

Comment:
MOVE_ISSUE?

Reply:
Yeah, I've added this before. It's for libultra, where the move pseudoinstruction gets assembled into the wrong thing. addu/addiu/or all can work, and it's doing the wrong one